src/sql/src/func.rs

```diff
@@ -2339,6 +2342,29 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         "pg_postmaster_start_time" => Scalar {
             params!() => UnmaterializableFunc::PgPostmasterStartTime => TimestampTz, 2560;
         },
+        "pg_relation_size" => Scalar {
+            params!(Any, String) => Operation::binary(|ecx, lhs, _| {
+                match ecx.scalar_type(&lhs) {
+                    ScalarType::Oid | ScalarType::RegClass => {}
+                    _ => sql_bail!("No function matches the given name and argument types. \
+                    You might need to add explicit type casts."),
+                }
```

Comment:
This isn't quite right. If this function has two separate versions that take two different argument types, it should have two separate `params!` listed. Alternatively, we might have a bug where we're not correctly implicitly casting between `oid` and `regclass` values. Either way, we shouldn't be switching on the scalar type inside the function's implementation!

Reply:
It's the later. Postgres only documents this function with `regclass`, and only has the two function oid's. But the tests with explicit oid parameters failed with `param!(RegClass)`. The conversion needs to work for dbeaver. Happy to change it, but I'll need guidance on how.